.github/CODEOWNERS

```diff
@@ -21,261 +21,276 @@
 
 * @open-telemetry/collector-contrib-approvers
 
-cmd/configschema/                                                       @open-telemetry/collector-contrib-approvers @mx-psi @dmitryax @pmcollins
```

Comment:
How are the spacing changes related to adding the distributions report? Are they necessary?

Reply:
no, they're not. Unfortunately the file was corrupted.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29727 is open to help fix some of this

Reply:
should be good there now, thanks!